fem/restriction.cpp

```diff
@@ -594,119 +595,246 @@ void L2ElementRestriction::FillJAndData(const Vector &ea_data,
    });
 }
 
+/// @brief Fills in the entries of the lexicographic face_map.
+///
+/// n_face_dofs_per_component is the number of DOFs for each vector component
+/// on the face (there is only one vector component in all cases except for 3D
+/// Nedelec elements, where the face DOFs have two components to span the
+/// tangent space).
+///
+/// The DOFs for the ith vector component begin at offsets[i] (i.e. the number
```

Comment:
`ith` -> `i-th`

Reply:
Fixed in 035301c212f0ab360d955b4d4fe2832fa3092f7f